modules/cms/classes/MediaLibrary.php

```diff
@@ -17,7 +17,7 @@ class MediaLibrary extends SystemMediaLibrary
      */
     protected function init()
     {
-        traceLog('Class Cms\Classes\MediaLibrary has been deprecated, use System\Classes\MediaLibrary instead.');
+        traceLog('Class Cms\Classes\MediaLibrary has been deprecated, use ' . SystemMediaLibrary::class . ' instead.');
```

Comment:
For consistency, would it not be best to also replace the first class name?
`traceLog('Class ' . static::class . ' has been deprecated, use ' . SystemMediaLibrary::class . ' instead.')`

Reply:
@36864, no, because if someone has extended Cms\Classes\MediaLibrary, then their message will be "Class Your\Custom\Class has been deprecated", which is not the case.

Reply:
`\Cms\Classes\MediaLibrary::class` then?

Reply:
Isn't this whole PR about pedantry? I just find it odd that you'd use `::class` for one thing and not another in the same line. And yes, `self::class` would be perfect here.

Reply:
I can fix that to `self::class` and yes I'm pedantic )